src/containers/landing.js

```diff
@@ -4,12 +4,36 @@ import { push } from 'react-router-redux';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
+import AdminClient from '../api/admin_client/index'
+
 import Button from '../components/Button';
 
 class Landing extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
+
+    this.createAdmin = this.createAdmin.bind(this);
+  }
+
+  createAdmin() {
+    // for testing
+
+    const payload = {
+      firstName: "the last",
+      lastName: "boob",
+      email: "boob29@boob.com",
+      username: "blah70",
+      password: "password"
+    }
+
+    AdminClient.getSingleton().createAdmin(payload)
+      .then(response => {
+        console.log('createAdmin', response);        
+      })
+      .catch(error => {
+        console.log('createAdmin failed', error);
+      })
```

Comment:
can we actually change this stuff to await syntax lmao

Reply:
all of these requests will be set up in reducers in the end so its up to whatever we decide/whoever writes the reducers.